File: Kth-Smallest-Element/KthSmallestElement.java
public class KthSmallestElement {

    public static void main(String[] args) {
        int arr[] = { 7, 10, 4, 3, 20, 15 };
        int n = arr.length;
        int k = 3;
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n - i - 1; j++) {
                if (arr[j] > arr[j + 1]) {
                    int temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                }
            }
        }
        System.out.println(arr[k - 1]);
    }
}

File: Kth-Smallest-Element/KthSmallestElement.js
function kthSmallestElement(arr, k) {
  let n = arr.length

  
  for (let i = 0; i < n; i++) {
    for (let j = 0; j < n - i - 1; j++) {
      if (arr[j] > arr[j + 1]) {
        let temp = arr[j]
        arr[j] = arr[j + 1]
        arr[j + 1] = temp
      }
    }
  }

  console.log(arr[k - 1])
}


let arr = [7, 10, 4, 3, 20, 15]
let k = 3
kthSmallestElement(arr, k)
